docs/resources/ocean_aks_np.md

```diff
@@ -306,4 +306,49 @@ update_policy {
     node_names = ["aks-omnp123456-7890-vmss000001"]
   }
 }
+```
+<a id="scheduled-task"></a>
+## Scheduled Task
+* `scheduled_task` - (Optional) An object used to specify times when the cluster will turn off. Once the shutdown time will be over, the cluster will return to its previous state.
+  * `shutdown_hours` - (Optional) An object used to specify times that the nodes in the cluster will be taken down.
+    * `is_enabled` - (Optional) Flag to enable or disable the shutdown hours mechanism. When False, the mechanism is deactivated, and the cluster remains in its current state.
+    * `time_windows` - (Required) The times that the shutdown hours will apply. Required if isEnabled is true.
+  * `tasks` - (Optional) A list of scheduling tasks to preform on the cluster at a specific cron time.
+    * `is_enabled` - (Required)  Describes whether the task is enabled. When true the task should run when false it should not run. Required for `cluster.scheduling.tasks` object.
+    * `cron_expression` - (Required) A valid cron expression. The cron is running in UTC time zone and is in Unix cron format Cron Expression Validator Script. Only one of `frequency` or `cronExpression` should be used at a time. Required for `cluster.scheduling.tasks` object. (Example: `0 1 * * *`).
+    * `task_type` - (Required) Valid values: `clusterRoll` The type of the scheduling task.
+    * `parameters` - (Optional) The parameters of the scheduling task. Each task type will have properties relevant only to it.
+      * `parameters_cluster_roll` - (Optional) The parameters of the cluster roll scheduling task.
+        * `batch_min_healthy_percentage` - (Optional) The minimum percentage of the scaled nodes that should be healthy at each batch. Valid values are 0-100.
+        * `batch_size_percentage` - (Optional) The percentage of the cluster that will be rolled at each batch. Valid values are 0-100.
+        * `comment` - (Optional) A comment to be added to the cluster roll.
+        * `respect_pdb` - (Optional) During the roll, if the parameter is set to true we honor PDB during the instance replacement.
+        * `respect_restrict_scale_down` - (Optional) During the roll, if the parameter is set to true we honor Restrict Scale Down label during the nodes replacement.
+        * `vng_ids` - (Optional) List of Virtual Node Group IDs to be rolled. If not set or set to null, cluster roll will be applied.
+
+```hcl
+scheduled_task {
```

Comment:
scheduled_task is not defined in the code, please modify

Reply:
updated to "scheduling"